packages/strapi-helper-plugin/lib/internals/generators/container/class.js.hbs

```diff
@@ -17,7 +17,7 @@ import { FormattedMessage } from 'react-intl';
 import { createStructuredSelector } from 'reselect';
 {{/if}}
 import { bindActionCreators, compose } from 'redux';
-// import pluginId from 'pluginId';
+import pluginId from '../../pluginId';
```

Comment:
Some specific generators should be added for the strapi-admin

Reply:
Can you explain what you've in mind?

Reply:
Well the admin doesn't need the plugin Id for instance, the selectors are a bit different and so on.
So if I find the time I'll submit a PR with some dedicated generators